src/player.cpp

```diff
@@ -11258,7 +11254,7 @@ bool player::can_study_recipe(const itype &book) const
         auto const r = elem.recipe;
         if( !knows_recipe( r ) &&
             ( !r->skill_used ||
-              get_skill_level( r->skill_used ) >= elem.skill_level ) ) {
+              ( get_skill_level( r->skill_used ) - elem.skill_level ) >= 4 ) ) {
```

Comment:
> factored out as a constant

Sure, though now that I think of it I think I'll factor out a method instead.
> and possibly reduced to 3?

Any particular rationale? I'm not saying 4 is perfect, it's just the middle of the previous random range, I'm just wondering if you have a reason.